run.sh

```diff
@@ -197,13 +197,27 @@ check_themes() {
       h3 "($i/$theme_count) '$theme_name' not found. Installing"
       WP theme install --quiet "$theme_name"
       STATUS $?
+      ((i++))
+      continue
+    fi
+
+    # Locally volumed themes
+    if [[ $theme_name =~ ^\[local\] ]]; then
```

Comment:
## BREAKING CHANGE

You are now required to explicitly mention all plugins and themes you depend on in the compose file, **including locally volumed plugins/themes!**

The format for locally volumed plugins/themes is `[local]my-plugin-slug` and `[local]my-theme-slug` (slug is the directory name by convention). 

This is required so that the build can still be aware that you need those when it goes in to look for orphaned plugins/themes. 

Without adding this to your docker-compose file, the build will think your volumed files are orphaned and it will delete them.... Full stop.... Repeat... _will delete them_.

@karellm if you have any better ideas as to how to go about this, I'm all ears. I can't think of a better way for this.